client/src/utils/navigation/navigation.yml

```diff
@@ -925,7 +925,7 @@ invocations:
     step_job_information: '[data-step="${order_index}"] .invocation-step-job-details .info_data_table'
     step_job_information_tool_id: '[data-step="${order_index}"] .invocation-step-job-details .info_data_table #galaxy-tool-id'
     export_tab: '.invocation-export-tab'
-    export_tab_disabled: '.invocation-export-tab .nav-link.disabled'
+    export_tab_disabled: '.invocation-export-tab .nav-link[aria-disabled="true"]'
```

Comment:
We likely expect `disabled` as class in other context, we should add `disabled` as class to the `GButton` instead.

Reply:
`aria-disabled` seems like a fairly standard thing compared to a `disabled` css class that has no semantic meaning ? If the selenium tests pass I think we shouldn't do that ?

Reply:
I think it's safer to use `disabled` instead, its standard for bootstrap and we have been using it universally for at least a decade, otherwise this can introduce subtle bugs.

Reply:
What subtle bugs would that be ? It's not safe to through random css classes into the mix just because "maybe"

Reply:
Ok, fair enough.